client/css/style.css

```diff
@@ -703,6 +708,33 @@ kbd {
 	transform: rotate(45deg) translateZ(0);
 }
 
+#sidebar .network .collapse-network {
+	border-radius: 3px;
+	width: 32px;
+	height: 34px;
+	float: left;
+	opacity: 0.4;
+	transition: opacity 0.2s, background-color 0.2s, transform 0.2s;
+}
+
+button.collapse-network:first-child:nth-last-child(3) {
```

Comment:
I can't tell if this is a total hack or brilliant :D Either way, probably the first time I see `:nth-last-child` actually useful so 👏, love it.

Reply:
(It's brilliant)